posts/2024/2024_06_28_Inigo_week_5.rst

```diff
@@ -23,7 +23,7 @@ Also, they provided their code in TensorFlow, so I started adapting it to our us
 What is coming up next week
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-I will continue working on the conditional AutoEncoder next week, but you can see the progress `here <https://github.com/itellaetxe/tractoencoder_gsoc/blob/main/src/tractoencoder_gsoc/models/cvae_model.py>`_.
+I will continue working on the conditional AutoEncoder next week, but you can see the progress `here <https://github.com/itellaetxe/tractoencoder_gsoc/blob/main/src/tractoencoder_gsoc/models/vae_model.py>`_.
```

Comment:
it does not look like a permalink. any change will break again this link.

Please, can you replace with this: 
```
https://github.com/itellaetxe/tractoencoder_gsoc/blob/56ddacd8f9a2c6ac30a38c1c141bf60cc1b2d862/src/tractoencoder_gsoc/models/vae_model.py
```

As you can see, the permalink contains the commit number which is important in this case. thanks for the future update

Reply:
Oh okay, I see now. Makes sense. Thank you. Updating.